core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -417,6 +418,33 @@ static function (array $uris): bool {
         ;
     }
 
+    private function addSanitizerNode(): NodeDefinition
+    {
+        return (new TreeBuilder('sanitizer'))
+            ->getRootNode()
+            ->addDefaultsIfNotSet()
+            ->children()
+                ->arrayNode('allowed_url_protocols')
+                    ->prototype('scalar')->end()
+                    ->defaultValue(['http', 'https', 'ftp', 'mailto', 'tel', 'data'])
```

Comment:
I would may be also add `skype` and `whatsapp` to the defaults, but otherwise 👍

Reply:
I agree 👍
Added in https://github.com/contao/contao/commit/15985ec5ac2b3b4540fb2ef0f6ecb1e1c3fa30f2.